build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:4.1.1'
+    classpath 'com.android.tools.build:gradle:3.6.1'
```

Comment:
Is this version compatible with Kotlin 1.4?

Reply:
Based on [this](https://docs.gradle.org/6.8/userguide/compatibility.html) it should be, but when I changed it to Kotlin 1.4 / plugin 3.6 / wrapper 6.8 it was not building / running (`ApkProvisionException: No outputs for the main artifact of variant: debug`)

We'll use a simpler version of Konfetti instead so that we can keep Kotlin 1.3 and plugin 3.6 compatibility (using wrapper 6.5 as listed in compatibility [here](https://docs.gradle.org/6.5/userguide/compatibility.html) instead of the 5.6.4 it was previously). I'll note this above too

How does this sound?

Reply:
Ack--thanks. I think keeping on Kotlin 1.4 sounds good since this is a nice chance to upgrade, anyway.